src/modules/core/components/Fields/InputStatus/InputStatus.tsx

```diff
@@ -1,6 +1,7 @@
 import { MessageDescriptor, useIntl } from 'react-intl';
 import React from 'react';
 
+import { isNil } from 'lodash';
```

Comment:
Could you please at least use import isNil from 'lodash/isNil' import? It will be less costly